src/lib/y2partitioner/widgets/pages/disks.rb

```diff
@@ -63,9 +64,12 @@ def contents
               )
             ),
             table,
-            HBox(
-              # TODO: add and edit need to be also added
-              DeviceDeleteButton.new(pager: pager, table: table)
+            Left(
+              HBox(
+                # TODO: add, edit and move buttons need to be also added
```

Comment:
Why not take the opportunity to add the missing buttons, as we are doing in most PBIs to avoid openQA headache.

Even more, add and edit should even work already. So the only dummy button we would be adding would be "move" (...which makes me think that maybe we should not add such button... let's see if nobody misses it).

Reply:
Well, we need to add some specific logic for those buttons. For example, when a partition is selected, the "add" button is meant to add a new partition over the device that the partition belongs to. And when a disk is selected, the "edit" button should jump to the disk page.

It is more or less easy to implement, but I would do it as a separate PR.

Reply:
Fine with me